SlicerCART/src/SlicerCART.py

```diff
@@ -82,21 +89,42 @@ def check_and_install_python_packages():
 IS_CLASSIFICATION_REQUESTED = True
 IS_SEGMENTATION_REQUESTED = True
 IS_SEMI_AUTOMATIC_PHE_TOOL_REQUESTED = True
+IS_MOUSE_SHORTCUTS_REQUESTED = False
+IS_KEYBOARD_SHORTCUTS_REQUESTED = True
 
 MODALITY = 'CT'
 
-LABEL_CONFIG_FILE_PATH = os.path.join(Path(__file__).parent.resolve(), "label_config.yml")
-KEYBOARD_SHORTCUTS_CONFIG_FILE_PATH = os.path.join(Path(__file__).parent.resolve(), "keyboard_shortcuts_config.yml")
-CLASSIFICATION_CONFIG_FILE_PATH = os.path.join(Path(__file__).parent.resolve(), "classification_config.yml")
-GENERAL_CONFIG_FILE_PATH = os.path.join(Path(__file__).parent.resolve(), "general_config.yml")
+REQUIRE_EMPTY = True
```

Comment:
Please split the code according to each issue since too many issues within the same pull request complicates the revision and acceptation process.

Reply:
What do you mean by "split the code"? Kuan already knows that there are too many issues addressed here, and in the future we will make sure to submit smaller changes per PR, but for now, it will take more time to re-do multiple PRs, than to just review this PR and merge it.

Reply:
for further follow-up, it would be a great idea to track the code change for each issue (ex now, some block of code added in the pull request refer to undisclosed issue --- if in 6 months we discover a new bug related to a specific issue, I anticipate it will be very complicated to track which part of the code is associated with this issue when referring to this pull request that was supposed to address it!

Reply:
For example of 'split the code', I mean ; download main branch --- create a branch for issue 40 --- add the code related to issue 40 --- submit a pull request --- Repeat for each issue addressed in this initial pull request. Total time estimated: 1 hour.

Reply:
Kuan and I will meet at the end of this afternoon (16h45) to address it!